docs/zkapps/testing-zkapps-lightnet.mdx

```diff
@@ -0,0 +1,233 @@
+---
+title: Testing zkApps with Lightnet
+hide_title: true
+description: Use lightweight Mina blockchain network (Lightnet) for testing on a local blockchain before you test with a live network. 
+keywords:
+  - zkApp CLI
+  - lightnet
+  - zkApp testing
+  - Docker image
+  - using zk lightnet
+---
+
+# Testing zkApps with Lightnet
+
+The most efficient way to deploy and run a lightweight Mina blockchain network for testing purposes is to spin up a lightweight Mina network (Lightnet) in a single Docker container. Lightnet is a resource-efficient solution with fast startup and syncing times.
+
+Lightnet lets you test your zkApp locally on an accurate representation of Mina blockchain before you test with a live network.
+
+Using Lightnet provides the following benefits:
+
+- Reduces the time from ideation to launch by letting you test your applications and changes against a close-to-real Mina network.
+- Starts a single-node network or a multi-node network with diverse types of participants.
+- Creates and funds accounts so that you can deploy and interact with your zkApp. The genesis ledger is configured with 1000 pre-funded accounts with a 1550 MINA balance on each account.
+- Optionally runs the Mina [archive node](/node-operators/archive-node).
+- Provides the Mina accounts manager helper tool so you can automate account retrieval.
+- Launches the lightweight Mina Explorer in a web browser so you can monitor transactions.
```

Comment:
These are repeated and should either be consistent with suggested changes in lightnet.mdx or removed and replaced with a reference to the lightnet section